okhttp-tests/src/test/java/okhttp3/CipherSuiteTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016 Google Inc.
```

Comment:
... I talked to our lawyers and they explained to me that this is fine. I’m thinking copyright works like property and you can assign us ownership here, and you can’t. So this is merely attribution. I’m cool with that, and I apologize for slowing this down and for getting this wrong.